src/org/usfirst/frc/team2706/robot/commands/autonomous/auto2018/automodes/CenterStartExchangeCube.java

```diff
@@ -0,0 +1,21 @@
+package org.usfirst.frc.team2706.robot.commands.autonomous.auto2018.automodes;
+
+import org.usfirst.frc.team2706.robot.commands.EjectCube;
+import org.usfirst.frc.team2706.robot.commands.InitLift;
+import org.usfirst.frc.team2706.robot.commands.autonomous.core.RotateDriveWithGyro;
+import org.usfirst.frc.team2706.robot.commands.autonomous.core.StraightDriveWithEncoders;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class CenterStartExchangeCube extends CommandGroup {
+    public CenterStartExchangeCube() {
+        this.addSequential(new InitLift());
+        this.addSequential(new StraightDriveWithEncoders(0.5, 46 / 12.0, 0.1, 3, this + ".driveForward"),2);
```

Comment:
I wonder if these could be constants or somehow mapped to elements of the field?